### PR TITLE
Feature 32 employee overview card

### PR DESCRIPTION
**Pull request by navina321**
created employee overview card with styling to match figma.

### CODE REVIEW

**Review by TristanPere — CHANGES_REQUESTED** [review]
Can you please make sure this branch is up to date with main. Before pull requests make sure to git pull origin main on your branch so it can be merged without conflicts.

File: src/components/EmployeeOverviewCard/EmployeeOverviewCard.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            {startDate}
+          </p>
+          <p className="employee-overview-card__right--course">
+            {courseCompletion}
+          </p>
+          <p className="employee-overview-card__right--manager">
+            {manager}
+          </p>
+        </div>
+      </div>
```
Comment (TristanPere) [inline src/components/EmployeeOverviewCard/EmployeeOverviewCard.jsx:34]: Make this a separate div.

File: src/components/EmployeeOverviewCard/EmployeeOverviewCard.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "./EmployeeOverviewCard.scss";
+import divide from "../../assets/images/functional-icons/overview-divide-icon.png";
+
+const EmployeeOverviewCard = ({ startDate, courseCompletion, manager, description }) => {
+  return (
+    <div className="employee-overview-card">
+      <h3 className="employee-overview-card__title">Employee Overview</h3>
+      <p className="employee-overview-card__text">    {description}
+      </p>
```
Comment (TristanPere) [inline src/components/EmployeeOverviewCard/EmployeeOverviewCard.jsx:10]: Make this the top div.

File: src/components/EmployeeOverviewCard/EmployeeOverviewCard.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          </p>
+          <p className="employee-overview-card__right--course">
+            {courseCompletion}
+          </p>
+          <p className="employee-overview-card__right--manager">
+            {manager}
+          </p>
+        </div>
+      </div>
+    </div>
```
Comment (TristanPere) [inline src/components/EmployeeOverviewCard/EmployeeOverviewCard.jsx:35]: The Figma card on close inspection is two divs inside a container where the two divs are separated by a line from edge to edge showing the background through it.

File: src/components/EmployeeOverviewCard/EmployeeOverviewCard.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import "./EmployeeOverviewCard.scss";
+import divide from "../../assets/images/functional-icons/overview-divide-icon.png";
+
+const EmployeeOverviewCard = ({ startDate, courseCompletion, manager, description }) => {
+  return (
+    <div className="employee-overview-card">
+      <h3 className="employee-overview-card__title">Employee Overview</h3>
+      <p className="employee-overview-card__text">    {description}
+      </p>
+      <img src={divide} alt="diving-line" className="employee-overview-card__divide"/>
```
Comment (TristanPere) [inline src/components/EmployeeOverviewCard/EmployeeOverviewCard.jsx:11]: Can remove this image using styling to show the gap between divs.

File: src/pages/Home/Home.jsx [hunk 5]
```diff
@@ -2,6 +2,7 @@ import Layout from "../../components/Layout/Layout";
 import NavContainer from "../../containers/NavContainer/NavContainer";
 import "../Home/Home.scss";
 
+
```
Comment (TristanPere) [inline src/pages/Home/Home.jsx:5]: remove white space

**Review by TristanPere — CHANGES_REQUESTED** [review]
The card has a great base just needs the separation of the two divs to match the figma

**Review by olirob93 — APPROVED** [review]

**Pull request merged**